datascroller/scroller.py

```diff
@@ -61,7 +60,7 @@ def update_dataframe_coords(self, start_row=0, start_col=0):
         self.r_2 = self.r_1 + self.rows_to_print
         self.c_2 = self.find_last_fitting_column()
 
-    def show_data_window_in_viewing_area(self):#, start_row=0, start_col=0):
+    def show_data_window_in_viewing_area(self):  # start_row=0, start_col=0):
```

Comment:
Looks like I left some stray comments in here as well. Optional to take care of now.